source/MaterialXCore/Element.cpp

```diff
@@ -64,13 +71,13 @@ bool Element::operator==(const Element& rhs) const
     }
 
     // Compare children.
-    const vector<ElementPtr>& c1 = getChildren();
```

Comment:
@kwokcb Does this PR contain any functional change to our `Element::operator==` method, or is this just a renaming of its variables to follow a new convention?  If it's the latter case, then I'd recommend that we omit this renaming work from the current PR, so that it's clear that this change is focused on new functionality, and doesn't change the behavior of existing methods.  After that, it should be fine to propose new naming conventions in a separate, non-functional PR.

Reply:
Sounds good. It was a search+replace change. I'll revert.